dietpi/dietpi-services

```diff
@@ -68,6 +69,8 @@
 			elif [ -f /etc/init.d/"${aSERVICE_NAME[$i]}" ]; then
 				aSERVICE_AVAILABLE[$i]=1
 
+			elif [ -f /etc/systemd/system/"${aSERVICE_NAME[$i]}".service ]; then
```

Comment:
yeah... this probably means that it won't work for on wheezy, but we really are supporting this except for legacy.  right?